src/media/js/previews.js

```diff
@@ -90,14 +90,17 @@ define('previews',
             // Initialize the bar position.
             updatePreviewBar($bars, slider);
 
-            // Add scroll buttons.
-            var buttons = previewButtons.attach(slider, $slider);
-
             if (document.documentElement.getAttribute('dir') === 'rtl') {
+                // Add scroll buttons.
+                var buttons = previewButtons.attach(slider, $slider, {rtl: true});
```

Comment:
hehe...building a hack on top of a hack. while i may have instigated this, i think it's time to turn around with a diff solution...